src/Database/Schema/PostgresSchema.php

```diff
@@ -191,27 +191,25 @@ protected function _defaultValue($default)
      */
     public function describeIndexSql($tableName, $config)
     {
-        $sql = 'SELECT
-            c2.relname,
-            a.attname,
-            i.indisprimary,
-            i.indisunique
-        FROM pg_catalog.pg_class AS c,
-            pg_catalog.pg_class AS c2,
-            pg_catalog.pg_index AS i,
-            pg_catalog.pg_attribute AS a
-        WHERE c.oid  = (
-            SELECT c.oid
-            FROM pg_catalog.pg_class c
-            LEFT JOIN pg_catalog.pg_namespace AS n ON n.oid = c.relnamespace
-            WHERE c.relname = ?
-                AND n.nspname = ?
-        )
-        AND c.oid = i.indrelid
-        AND c.oid = a.attrelid
-        AND a.attnum = ANY(i.indkey)
-        AND i.indexrelid = c2.oid
-        ORDER BY i.indisprimary DESC, i.indisunique DESC, c2.relname, a.attnum';
+        $sql = "SELECT
+            c.conname AS name,
+            c.contype AS type,
+            a.attname AS column_name,
+            c.confmatchtype AS match_type,
+            c.confupdtype AS on_update,
+            c.confdeltype AS on_delete,
+            c.confrelid::regclass AS references_table,
+            ab.attname AS references_field
+        FROM pg_catalog.pg_namespace n
+        INNER JOIN pg_catalog.pg_constraint c ON (n.oid = c.connamespace)
```

Comment:
I don't think this will work. You are using pg_constraint to get index data which won't be there.